app/controllers/discussions_controller.rb

```diff
@@ -183,6 +183,16 @@ def update_version
     redirect_to @version.reify()
   end
 
+  def print
+    @discussion = Discussion.find_by_key! params[:id]
```

Comment:
This is way too many queries just to access discussion relations. Can we use `includes` to minimize this?

```
@discussion = Discussion.includes(:group, :motions, :items).find_by_key(params[:id])
``` 

Then you can safely do `@discussion.items` within the view without incurring another db query.